<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SidebarUi</class>
 <widget class="QWidget" name="SidebarUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>248</width>
        <height>423</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Model Selection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="selectModel"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>223</width>
        <height>461</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Affine Transformations</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Translation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="horizontalSpacing">
           <number>15</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="xTranslation">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="yTranslation">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="zTranslation">
            <property name="minimum">
             <number>-100</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="xTranslationSpin"/>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="yTranslationSpin"/>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="zTranslationSpin"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Rotation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="horizontalSpacing">
           <number>15</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <item row="2" column="2">
           <widget class="QSpinBox" name="yRotationSpin"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Pitch</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="rRotationSpin"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Roll</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Yaw</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="pRotationSpin"/>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="rRotation">
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="pRotation">
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="yRotation">
            <property name="minimum">
             <number>-360</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Scale</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="horizontalSpacing">
           <number>15</number>
          </property>
          <property name="verticalSpacing">
           <number>2</number>
          </property>
          <item row="3" column="1">
           <widget class="QSlider" name="zScale">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QSpinBox" name="uScaleSpin"/>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="yScale">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>All</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSlider" name="uScale">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSlider" name="xScale">
            <property name="maximum">
             <number>10</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="zScaleSpin"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="yScaleSpin"/>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="xScaleSpin"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <attribute name="label">
       <string>View Transformations</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="page_4">
      <attribute name="label">
       <string>Render Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Display Mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="smooth">
            <property name="text">
             <string>Smooth</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="flat">
            <property name="text">
             <string>Flat</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="wireframe">
            <property name="text">
             <string>Wire</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="autoExclusive">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Additional Display Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="showNormals">
            <property name="text">
             <string>Show Normals</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
